### PR TITLE
Allow specify playback range for animation state

### PR DESCRIPTION
**Pull request by shrinktofit**
Re: cocos-creator/3d-tasks#

Changelog:
 *

### CODE REVIEW

File: cocos/3d/skinned-mesh-renderer/skinned-mesh-renderer.ts [hunk 1]
```diff
@@ -142,7 +142,7 @@ export class SkinnedMeshRenderer extends MeshRenderer {
     private _updateModelType () {
         if (!this._skinningRoot) { return; }
         const comp = this._skinningRoot.getComponent('cc.SkeletalAnimation') as SkeletalAnimation;
-        if (comp) { this.setUseBakedAnimation(comp.useBakedAnimation); }
+        if (comp) { this.setUseBakedAnimation(comp.useBakedAnimation); } else { this.setUseBakedAnimation(false); }
```
Comment (shrinktofit) [inline cocos/3d/skinned-mesh-renderer/skinned-mesh-renderer.ts:145]: @YunHsiao

Comment (YunHsiao) [reply cocos/3d/skinned-mesh-renderer/skinned-mesh-renderer.ts:145]: LGTM

File: cocos/core/animation/animation-state.ts [hunk 2]
```diff
@@ -268,6 +269,29 @@ export class AnimationState extends Playable {
...
+     * 获取或设置播放范围。
+     * 范围的 `min`、`max` 字段都是以秒为单位的。
+     * 设置时，应当指定一个有效的范围；实际的播放范围是该字段和 [0, 周期] 之间的交集。
+     * 设置播放范围时将重置累计播放时间。
+     */
+    get playbackRange (): Readonly<{ min: number; max: number; }> {
+        return this._playbackRange;
+    }
+
+    set playbackRange (value) {
```
Comment (gameall3d) [inline cocos/core/animation/animation-state.ts:288]: why not add type for value?

**Review by gameall3d — CHANGES_REQUESTED** [review]

**Review by gameall3d — APPROVED** [review]
LGTM

**Review by SantyWang — APPROVED** [review]

**Pull request merged**